php7.1/apache/php-test-runner.ini

```
@@ -0,0 +1,2 @@
+; Add custom php.ini configuration here
+error_reporting=E_ALL
```

```suggestion
error_reporting=E_ALL

```
Just adding the new line.